index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog | Seattle's Premier Dog Adoption Agency</title>
+    <script src="https://kit.fontawesome.com/4b01e0c907.js"></script>
+    <link rel="stylesheet" href="styles/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Serif:400,400i&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <p><a href="#main-content" class="skip-link">Skip to main content</a></p>
```

Comment:
I'm guessing this was for testing purposes initially but now is messing with the layout of the site. 
I would either remove the element or set display to false. Simply moving it to the left -10000 px does not remove the impact it has on vertical shifting of elements.